### ISSUE TITLE
lambda-term.1.12.0 doesn't compile on 4.02.3

### ISSUE THREAD

**Issue by jjd27** [ocaml/opam-repository#10635]
```#=== ERROR while installing lambda-term.1.12.0 ================================#
# opam-version 1.2.2
# os           linux
# command      jbuilder build -p lambda-term -j 4
# path         /home/jonathandav/.opam/4.02.3/build/lambda-term.1.12.0
# compiler     4.02.3
# exit-code    1
# env-file     /home/jonathandav/.opam/4.02.3/build/lambda-term.1.12.0/lambda-term-23227-c266f8.env
# stdout-file  /home/jonathandav/.opam/4.02.3/build/lambda-term.1.12.0/lambda-term-23227-c266f8.out
# stderr-file  /home/jonathandav/.opam/4.02.3/build/lambda-term.1.12.0/lambda-term-23227-c266f8.err
### stderr ###
# Warning 3: deprecated: Lwt_unix.execute_job
# [...]
#     ocamlopt src/lTerm_resource_lexer.{cmx,o}
#     ocamlopt src/lTerm_mouse.{cmx,o}
#       ocamlc src/lTerm.{cmi,cmti}
#       ocamlc src/lTerm_draw.{cmo,cmt}
#       ocamlc src/lTerm_unix.{cmo,cmt} (exit 2)
# (cd _build/default && /home/jonathandav/.opam/4.02.3/bin/ocamlc.opt -w -40 -safe-string -g -bin-annot -I /home/jonathandav/.opam/4.02.3/lib/bytes -I /home/jonathandav/.opam/4.02.3/lib/camomile -I /home/jonathandav/.opam/4.02.3/lib/lwt -I /home/jonathandav/.opam/4.02.3/lib/lwt_react -I /home/jonathandav/.opam/4.02.3/lib/ocaml -I /home/jonathandav/.opam/4.02.3/lib/react -I /home/jonathandav/.opam/4.02.3/lib/result -I /home/jonathandav/.opam/4.02.3/lib/zed -no-alias-deps -I src -o src/lTerm_unix.cmo -c -impl src/lTerm_unix.ml)
# File "src/lTerm_unix.ml", line 342, characters 32-51:
# Error: This expression has type bytes but an expression was expected of type
#          string
```

**Comment by yuyube23** [comment]
I got the same error; I ended up doing `opam install lambda-term.1.11`, then `opam install utop`, and that seem to have worked.

**Comment by bratan05** [comment]
Same issue there. Solved by what yuyube23 suggested.

**Comment by ncihnegn** [comment]
This has been fixed in 1.13 release.
